common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -209,7 +214,7 @@ def get_real_social_auth_object(request):
     if running_pipeline and 'social' in running_pipeline['kwargs']:
         social = running_pipeline['kwargs']['social']
         if isinstance(social, dict):
-            social = UserSocialAuth.objects.get(uid=social.get('uid', ''))
+            social = social_django.models.UserSocialAuth.objects.get(uid=social.get('uid', ''))
```

Comment:
@bradenmacdonald Will fix this currently by passing in `**social` kwargs. Right now, replacement would mean affecting edx-enterprise as well and writing more code, which I guess we're a little short on time for at this stage.